source/_integrations/hive.markdown

```diff
@@ -23,43 +23,42 @@ This integration uses the unofficial API used in the official Hive website [http
 
 There is currently support for the following platforms within Home Assistant:
 
-- [Binary Sensor](#binary-sensor)
-- [Climate](#climate)
-- [Light](#light)
-- [Sensor](#sensor)
-- [Switch](#switch)
-- [Water Heater](#water-heater)
+- [Configuration](#configuration)
+- [Options](#options)
+- [Services](#services)
+  - [Service `hive.boost_heating`](#service-hiveboost_heating)
+  - [Service `hive.boost_hot_water`](#service-hiveboost_hot_water)
+- [Platforms](#platforms)
+  - [Binary Sensor](#binary-sensor)
+  - [Climate](#climate)
+  - [Light](#light)
+  - [Sensor](#sensor)
+  - [Switch](#switch)
+  - [Water Heater](#water-heater)
```

Comment:
Something goes wrong here, possibly this list can be completely removed due to the TOC on the side of the page.

Reply:
Hi @klaasnicolaas  this has been removed in the latest commit